src/lib.rs

```diff
@@ -119,7 +135,9 @@ pub fn inaccurate_sort(strings: &mut Vec<String>) {
     strings.sort_by(|a, b| str_cmp(&a, &b));
 }
 
-pub fn assign_commits(log: &slog::Logger, release_name: &str, previous_release: &str, path: &str) {
+pub fn assign_commits(log: &slog::Logger, release_name: &str, previous_release: &str, release_project_id: i32, path: &str) {
+    // Could take the connection as a parameter, as problably
+    // it's already established somewhere...
```

Comment:
yeah; i'm totally okay with doing this this way for now; i want to clean up all the connection stuff. we'll get there.